### DISCUSSION TITLE
pluto grid 캘린더 관련 질문

### DISCUSSION THREAD

**Discussion by fenoteryn** [bosskmk/pluto_grid#136] [Q&A]
date type의 셀을 수정할 때 나타나는 캘린더가 날짜만 표시되어 있고 월은 표시 안 되어 있어서 몇 월 며칠을 나타내는지 모르겠습니다. 월을 나타낼 수 있는 방법이 있을까요? 그리고 그리드 안에서 column을 검색할 때 언제부터~언제까지의 날짜를 검색하는 방법이 있나요?

**Comment by bosskmk** [comment]
날짜 선택시 월 표시 : 
> 현재 선택 된 날짜의 월이 상단에 표시 되고 있습니다. 그리드에서 날짜 선택팝업은 개선의 여지가 많이 있습니다. 공휴일의 빨간색 표시나 월 단위로 페이징을 하는 등...

날짜 범위 검색 :
> 필터 설정에서 조건을 두개 설정 하면 됩니다. Greater than or equal to ~ Less than or equal to

**Reply by fenoteryn** [reply]
한 column에 두 개의 조건을 설정하는 방법이 있나요?

**Reply by bosskmk** [reply]
필터 설정(Set filter) 메뉴를 탭하면 나오는 팝업에서 여러개의 조건을 설정 할 수 있습니다.
